ouroboros-network/src/Ouroboros/Network/Protocol/LocalTxSubmission/Test.hs

```diff
@@ -0,0 +1,232 @@
+{-# LANGUAGE NamedFieldPuns            #-}
+{-# LANGUAGE FlexibleInstances          #-}
+{-# LANGUAGE GADTs                      #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE RankNTypes                 #-}
+{-# LANGUAGE KindSignatures             #-}
+{-# LANGUAGE PolyKinds                  #-}
+{-# LANGUAGE DataKinds                  #-}
+
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Ouroboros.Network.Protocol.LocalTxSubmission.Test (
```

Comment:
Not a big deal but I realised that we use `Tests` in `typed-protocols` and `Test` in `ouroboros-network`.

Reply:
True, could tidy that up later.